Automattic/jetpack
# Subscribe block: do not show in page editor and for private sites

Fixes #23688

#### Changes proposed in this Pull Request:

* Subscriptions are only available for posts, and not for private sites on WoA, so let's add some conditions to the display of subscription panels in the block editor.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Test on a WoA site, connected to WordPress.com, still private (not launched).
* Go to Jetpack > Settings and enable the Subscriptions feature.
* Go to Pages > Add New, type some content.
* Save your draft, and hit the Publish button; you should not see the Subscriptions panel in the Pre-publish panel.
* Now repeat the same thing when creating a post.
* Now repeat the same thing after setting the site to public; you should see the Subscriptions panel now.
